server/models/Expense.js

```diff
@@ -358,6 +358,7 @@ function defineModel() {
         ]),
         host: get(host, 'minimal'),
         collective: { ...this.collective.minimal, isActive: this.collective.isActive },
+        path: this.collective.getUrlPath(),
```

Comment:
I personally think we should address this globally rather than patching emails individually. If we do want to patch them individually, then we need to figure out a better variable name: I don't think anyone will guess that `data.path` will be set to the collective URL when looking at an `expense.missing.receipt` activity.

Reply:
@Betree : I've renamed it to `urlPath` to make it explicit and also added with within the `collective` object. This way I think it's clearer. 

As for a global solution, I've opened a new issue and will work on it; https://github.com/opencollective/opencollective/issues/5979